packages/config/config/devices/0x0344/he-rs01.json

```diff
@@ -18,51 +18,8 @@
 			"label": "Frequency Region",
 			"valueSize": 1,
 			"minValue": 0,
```

Comment:
Why are we removing the options? Even if read only the UI will display those for the corresponding value (or it should).

Reply:
I've brought them back. Was a mistake

Reply:
Just to give some context: UIs can use the options to show nice labels for the values, even if the parameters are readonly.

Reply:
I don't see the options anymore. We should keep them even when readonly, because UIs will display the label instead of numbers.

Reply:
Done